authoringAssistant/learningCases/methane-category-perc.fld

```diff
@@ -14,3 +14,4 @@ let getByCategory category year data =
       in helper data;
 let year = 2030;
 let record = getByCategory "$emissions.source$" $emissions.year$ tableData
+in
```

Comment:
It may make more sense to keep the `in` out of these files and think of them as more like “modules” (what we call a “module” is really just a bunch of `let` expressions without the trailing `in`). Let’s pick this up as part of #80.